src/pages/index.js

```diff
@@ -81,45 +145,110 @@ const newCardPopup = new PopupWithForm({
 });
 newCardPopup.setEventListeners();
 
-const profileEditPopup = new PopupWithForm({
-  popupSelector: "#profile-edit-modal",
-  handleFormSubmit: (data) => {
-    handleProfileFormSubmit(data);
-  },
+// const avatarEditPopup = new PopupWithForm({
+//   popupSelector: "#update-avatar-modal",
+// });
+// avatarEditPopup.setEventListeners();
+
+const deleteCardPopup = new PopupWithForm({
+  popupSelector: "#delete-card-modal",
 });
-profileEditPopup.setEventListeners();
+deleteCardPopup.setEventListeners();
 
 const userInfo = new UserInfo({
   profileTitleSelector: ".profile__title",
   profileDescriptionSelector: ".profile__description",
+  avatarSelector: ".profile__image",
+});
+api
+  .getUserInfo(userInfo._title, userInfo._description)
```

Comment:
Better to use `Promise.all` to load initial data

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all